src/js/models/figure_model.js

```diff
@@ -550,9 +550,14 @@
         },
 
         getDefaultFigureName: function() {
+            const padL = (nr, len = 2, chr = `0`) => `${nr}`.padStart(2, chr);
```

Comment:
`len` argument isn't used in `padStart()`, but you only ever use the first argument anyway, so you could remove the `len` and `chr` args?

Reply:
Sorry for that, corrected in the last commit